src/components/Title.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import style from '../../styles/Title.css';
+
+// props w nawiasie? chyba bez różnicy
```

Comment:
this is an arrow function - when you have only one argument in this type of function brackets are unnecessary ;)

comments in english ;)